packages/test/end-to-end-tests/src/test/oldVersion.ts

```diff
@@ -158,7 +158,10 @@ export class LocalTestObjectProvider<TestContainerConfigType> {
     public async makeTestContainer(testContainerConfig?: TestContainerConfigType) {
         const loader = this.makeTestLoader(testContainerConfig);
         const container =
-            await createAndAttachContainer(defaultDocumentId, defaultCodeDetails, loader, this.urlResolver);
+            await createAndAttachContainer(defaultCodeDetails, loader, {
+                url: `http://localhost:3000/${defaultDocumentId}`,
+                headers: { createNew: true },
```

Comment:
i'm seeing this in `contextReload.spec.ts` instead:
`urlResolver.createCreateNewRequest(documentId)`